new-package/github/workflows/test.yml

```diff
@@ -64,23 +67,26 @@ jobs:
   publish:
     needs: [test] # make sure all your other jobs succeed before trying to publish
 
+    permissions:
+      contents: write
+
```

Comment:
I read through the `dillonkearns/elm-publish-action` source code, and I think this is all it needs.

Reply:
Alright. Next time I try to publish a package I'll try to backport this change there and test it out.